utils/webhook-signature.ts

```diff
@@ -0,0 +1,101 @@
+import { createHmac } from "crypto";
+import { isNaN } from "lodash";
+import type { WebhookEvent } from "../libs/interfaces";
+
+class WebhookSignatureError extends Error {
```

Comment:
Refactor this class to `libs/error/WebhookSignatureError.ts` and export it

so it can be imported on user side as `import { WebhookSignatureError } from 'imagekit/dist/libs/errors';`